src/client/pages/Offer/InsuranceSelector/InsuranceSelector.helpers.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  QuoteBundleVariant,
+  ExternalInsuranceDataQuery,
+  InsuranceDataCollection,
+} from 'data/graphql'
+
+/**
+ * Only Car is supported
+ */
+export const getInsuranceExposure = (variants: QuoteBundleVariant[]) => {
+  const registrationNumber =
```

Comment:
Odd syntax, could it be `.registrationNumber` or destructuring?

Reply:
Yeah sure, I just felt like I wanted to highlight that we are accessing an untyped